lib/cn/cn_perfc.c

```diff
@@ -18,26 +19,28 @@
 /* clang-format off */
 
 struct perfc_name cn_perfc_get[] _dt_section = {
-    NE(PERFC_RA_CNGET_GET,       2, "Count of cN lookup attempts",   "c_get(/s)"),
-    NE(PERFC_RA_CNGET_MISS,      2, "Count of cN lookup misses",     "c_mis(/s)"),
-    NE(PERFC_RA_CNGET_TOMB,      2, "Num lookups that found tombs",  "c_tmb(/s)"),
+    NE(PERFC_RA_CNGET_MISS,      2, "cN lookup miss rate",           "c_mis(/s)"),
```

Comment:
The prefix 'r' probably means rate. What's 'c' for?

Reply:
Ah, probably count. But if they're all described as rate now maybe we can change them to r.

Reply:
Right, I didn't want to change the existing ones for fear of breaking grafana.  But I want to come back with a different patch and fix them all, but need to coordinate with @twblamer to do that...